ramls/request-preference.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request preference schema",
+  "properties": {
+    "id": {
+      "description": "Unique request preference ID",
+      "type": "string"
+    },
+    "userId": {
+      "description": "UUID of user associated with this request preference",
+      "type": "string"
+    },
+    "holdShelf": {
+      "description": "Whether 'Hold shelf' option is available to the user.",
```

Comment:
Ah, I think I misunderstood this. My understanding was that its whether "holdShelf" or "delivery" option was selected by the user - not whether which of these options is available to user.